pkg/beacon/relay/gjkr/integration_test.go

```diff
@@ -904,16 +904,12 @@ func TestExecute_DQ_member2_notRevealsDisqualifiedQualMemberKey_phase11(t *testi
 	seed := dkgtest.RandomSeed(t)
 
 	interceptor := func(msg net.TaggedMarshaler) net.TaggedMarshaler {
-		// Member 1 misbehaves by sending an accusation with wrong private key.
+		// Member 1 misbehaves by sending an invalid public key share points message.
 		// As result, they should be disqualified by other members.
-		accusationsMessage, ok := msg.(*gjkr.PointsAccusationsMessage)
-		if ok && accusationsMessage.SenderID() == group.MemberIndex(1) {
-			randomKeyPair, _ := ephemeral.GenerateKeyPair()
-			accusationsMessage.SetAccusedMemberKey(
-				group.MemberIndex(2),
-				randomKeyPair.PrivateKey,
-			)
-			return accusationsMessage
+		publicKeyShareMessage, ok := msg.(*gjkr.MemberPublicKeySharePointsMessage)
+		if ok && publicKeyShareMessage.SenderID() == group.MemberIndex(1) {
+			publicKeyShareMessage.RemovePublicKeyShare(2)
+			return publicKeyShareMessage
```

Comment:
I don't think I follow... why did we have to modify this test?

Reply:
Here we are testing the following scenario:
1. Member 1 misbehaves, doesn't matter how
2. Member 2 doesn't reveal member 1 as misbehaving before the reconstruction phase. As a result, member 2 should be disqualified.

Until now, member 1 misbehaved by sending an invalid `PointsAccusationsMessage` thus became DQ in phase 9. Member 2 should reveal them as DQ before the reconstruction phase. But, because we changed the logic of `memberForReconstruction` and members marked IA/DQ in phase 9 are no longer selected for reconstruction if they provided valid public key shares in phase 8, this test started to fail because member 2 stopped to reveal member 1 as expected. Because of that, I changed the moment when member 1 becomes DQ in the way it remains selected for reconstruction.

Reply:
But shouldn't we mark member 1 as disqualified if they send accusation with wrong private key?

Reply:
OK, we do mark member 1 as disqualified. We had to change this logic so that we mark member 2 as disqualified and it makes perfect sense.

What we are testing here is whether member 2 should be disqualified. And given the current logic, member 2 is not required to reveal ephemeral public key of member 1, if that member sent and invalid `PointsAccusationMessage` in phase 9.

The change Lukasz made aims at disqualifying member 1 earlier, so that member 2 is required to reveal ephemeral key and it's not doing that - this is the behaviour we are stressing here.